sonar-text-plugin/build.gradle.kts

```diff
@@ -0,0 +1,75 @@
+plugins {
+  id("org.sonarsource.text.java-conventions")
+  id("org.sonarsource.text.artifactory-configuration")
+  id("com.github.johnrengelman.shadow") version "8.1.1"
```

Comment:
Please extract version

Reply:
I tried but failed :-(

Reply:
Yes, `plugins {}` block is treated differently than the rest of the scripts. There is support for `[plugins]` section in TOML catalog; then these plugins can be accessed with `alias(libs.plugins...)`